Gemfile

```diff
@@ -5,6 +5,7 @@ ruby '~> 2.6.5'
 
 gem 'rails', '~> 6.0.0'
 
+gem 'autoprefixer-rails', "~> 10.0"
```

Comment:
We probably should have been using this all along:

https://github.com/uswds/uswds#sass-compilation-requirements